cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```
@@ -456,6 +482,9 @@ objects:
         - name: azure-credentials
           secret:
             secretName: azure-credentials
+        - name: arm-identity
+          secret:
+            secretName: arm-identity
```

```suggestion
          csi:
            driver: secrets-store.csi.k8s.io
            readOnly: true
            volumeAttributes:
              secretProviderClass: arm-identity
```
This is not a secret per se. It comes from the CSI driver, and the configuration is a bit different from what we do on the CS project.